app/javascript/components/Groceries/Groceries.tsx

```diff
@@ -55,6 +55,7 @@ class Groceries extends React.Component<RouteComponentProps, GroceriesState> {
     this.modifyItemInList = this.modifyItemInList.bind(this);
     this.handleCreateOrder = this.handleCreateOrder.bind(this);
     this.addNewItemToBasket = this.addNewItemToBasket.bind(this);
+    this.cartItemPrice = this.cartItemPrice.bind(this);
```

Comment:
should be verb (e.g. `getCartTotalPrice`)

Reply:
done